tests/test_distributed/test_with_spark/test_spark_local.py

```diff
@@ -158,6 +159,47 @@ def reg_with_weight(
     )
 
 
+RegData = namedtuple("RegData", ("reg_df_train", "reg_df_test"))
+
+
+@pytest.fixture
+def reg_data(spark: SparkSession) -> Generator[RegData, None, None]:
```

Comment:
Do we need tests for multi-class as well?

Reply:
Yeah, good suggestion.